hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/json/GroupConditionsInfo.java

```diff
@@ -35,7 +39,11 @@
  * @author lucas ponce
  */
 public class GroupConditionsInfo {
+
+    @JsonInclude(Include.NON_EMPTY)
```

Comment:
just leave it, i was only noting that it would likely not be empty given the purpose of the utility class.